README.md

```diff
@@ -14,11 +14,13 @@ To use the payment service, you need to sign up for a Paytrail account. Transact
 ### General requirements
 
 - PHP version >= 7.3
-- [Guzzle](https://github.com/guzzle/guzzle) 7 or 6 - PHP HTTP client for performing HTTP request.
```

Comment:
Should Guzzle still be recommended even if it isn't strictly required?

Reply:
Yes please. curl sometimes suck hard when unexpected issues (network etc) occurs, giving no info whatsoever about what the error is.

Reply:
Added recommendation to readme.